### DISCUSSION TITLE
Update howto for manual installation

### DISCUSSION THREAD

**Discussion by zoeperman** [SBFspot/SBFspot#547] [General]
Hi,

Is there a how-to on how to manual update on een linux VM?
I'm running ver 3.8 and want to update to the latest version but can't figure out without data loss (or config loss)

**Comment by sbf-** [comment]
Backup config and database before upgrade. It's always safe to have a backup ;-)

**Reply by zoeperman** [reply]
true, but what actions need to be done in what order to update

**Reply by kayD** [reply]
Also save your crontab, as it will be overwritten (learned that the hard way :cowboy_hat_face: )

**Reply by zoeperman** [reply]
> Backup config and database before upgrade. It's always safe to have a backup ;-)

Is it possible to make a how to for it?

**Reply by sbf-** [reply]
Build SBFspot and SBFspotUpload following the wiki, but **_without_** the `make install_sqlite` (Running the install command will overwrite the db and config)
Stop the upload daemon: `sudo systemctl stop SBFspotUpload.service`
Copy the new SBFspot and daemon to `/usr/local/bin/sbfspot.3`
Restart the daemon: `sudo systemctl start SBFspotUpload.service`
In theory there is no need to stop the cron job if you copy SBFspot just between two runs.

As you can see, it's zoeper easy :-)

**Reply by zoeperman** [reply]
I think i did something wrong in this part..
_Copy the new SBFspot and daemon to /usr/local/bin/sbfspot.3_
because al the config files where overwritten bij defaults
i did this, but i think you did mean something else ;)

sudo cp -r /home/zoeperman/SBFspot/SBFspot/* /usr/local/bin/sbfspot.3/
sudo cp -r /home/zoeperman/SBFspot/SBFspotUploadDaemon/* /usr/local/bin/sbfspot.3/

because i still get this:
/usr/local/bin/sbfspot.3/SBFspot -v -finq -nocsv
SBFspot **V3.8.0**
